### PR TITLE
Improve readme.md

### LINKED ISSUE #152: improve Readme.md

### PR DESCRIPTION
**Pull request by Elli610**
closes #152

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,9 @@
 This repository is a collection of tools, objects and functions in *AssemblyScript*.
 
 - *Classes and helper functions* with **as-types**
```
Comment (0xSwapFeeder) [inline README.md:9]: Helper functions but also SC types and type serialization ?

File: README.md [hunk 2]
```diff
@@ -7,7 +7,9 @@
...
-- *random draws* with **as-proba**
+
+- *Random draws* with **as-proba**
```
Comment (0xSwapFeeder) [inline README.md:11]: Not just random draws ? Maybe just precise what his purpose is for SC developers ?

File: README.md [hunk 3]
```diff
@@ -7,7 +7,9 @@
...
+
 - *AssemblyScript transformers for unit testing and reading a file* with **as-transformer**
```
Comment (0xSwapFeeder) [inline README.md:13]:
```suggestion
- *AssemblyScript transformers, dynamic code replacement features* with **as-transformer**
```

File: README.md [hunk 4]
```diff
@@ -51,3 +53,30 @@ npm run test
...
+## Contributing
+We welcome contributions from the community!
+
+If you would like to contribute to Massa-as-sdk, please read the [CONTRIBUTING file](CONTRIBUTING.md).
+
+## License
+Massa-as-sdk is released under the [MIT License](LICENSE).
+
+## Powered By
+Massa-as-sdk is developed with love by MassaLabs and powered by a variety of [open-source projects](powered-by.md).
```
Comment (0xSwapFeeder) [inline README.md:82]: Bad copy/paste ?

File: README.md [hunk 5]
```diff
@@ -51,3 +53,30 @@ npm run test
...
+	generateEvent(`The result of the combination of ${n.toString()} and ${k.toString()} is ${result.toString()}`);
+}
+```
+## Contributing
+We welcome contributions from the community!
+
+If you would like to contribute to Massa-as-sdk, please read the [CONTRIBUTING file](CONTRIBUTING.md).
+
+## License
+Massa-as-sdk is released under the [MIT License](LICENSE).
```
Comment (0xSwapFeeder) [inline README.md:79]: Bad copy/paste ?

File: README.md [hunk 6]
```diff
@@ -51,3 +53,30 @@ npm run test
...
+	const  n = args.nextU64();
+	const  k = args.nextU64();
+	const  result = combination(n, k);
+	generateEvent(`The result of the combination of ${n.toString()} and ${k.toString()} is ${result.toString()}`);
+}
+```
+## Contributing
+We welcome contributions from the community!
+
+If you would like to contribute to Massa-as-sdk, please read the [CONTRIBUTING file](CONTRIBUTING.md).
```
Comment (0xSwapFeeder) [inline README.md:76]: Bad copy/paste ?

**Review by 0xSwapFeeder — CHANGES_REQUESTED** [review]
Need improvement

File: README.md (hunk 5 above)
Comment (Elli610) [reply README.md:79]: We havent created this files yet. Once we do it it will work

File: README.md (hunk 4 above)
Comment (Elli610) [reply README.md:82]: We havent created this files yet. Once we do it it will work

File: README.md (hunk 6 above)
Comment (Elli610) [reply README.md:76]: We havent created this files yet. Once we do it it will work

File: README.md (hunk 2 above)
Comment (Elli610) [reply README.md:11]: What do you think about "*Most essential Classes and helper functions* with **as-types**" ?

File: README.md (hunk 4 above)
Comment (0xSwapFeeder) [reply README.md:82]: Not talking about the link the the fact that you are talking about Massa-as-sdk

File: README.md (hunk 5 above)
Comment (0xSwapFeeder) [reply README.md:79]: same

File: README.md (hunk 6 above)
Comment (0xSwapFeeder) [reply README.md:76]: same

File: README.md (hunk 2 above)
Comment (0xSwapFeeder) [reply README.md:11]: > What do you think about "_Most essential Classes and helper functions_ with **as-types**" ?

For the first comment, but I agree.

Comment (0xSwapFeeder) [reply README.md:11]: What about: "*Probabilistic simulation functionalities for smart contracts*" ?

**Review by 0xSwapFeeder — CHANGES_REQUESTED** [review]
Discussing in comments.

**Review by 0xSwapFeeder — CHANGES_REQUESTED** [review]

**Review by 0xSwapFeeder — APPROVED** [review]
lgtm :+1:

File: README.md [hunk 7]
```diff
@@ -51,3 +53,30 @@ npm run test
...
 ```
+## Usage
+After installing *As*, you can import the object classes and functions that you need in your AssemblyScript file.
+
+For example, to use the `combination` function to determine the combination of 2 numbers, you can import and use it like this:
+```typescript
+import { combination } from  '@massalabs/as-proba';
+import { Args } from  '@massalabs/as-types';
+
+// This main function is called automatically when the smart contract is executed by the blockchain.
```
Comment (Ben-Rey) [inline README.md:64]: I'm not sure it's still true

File: README.md [hunk 8]
```diff
@@ -51,3 +53,30 @@ npm run test
...
+//the argument args contains the serialized values n and k (n >= k) as U64
+export function main(args: StaticArray<u8>): void {
```
Comment (Ben-Rey) [inline README.md:66]: I would use something more specific than main like calculateCombination

File: README.md [hunk 9]
```diff
@@ -6,9 +6,11 @@
 
 This repository is a collection of tools, objects and functions in *AssemblyScript*.
 
-- *Classes and helper functions* with **as-types**
-- *random draws* with **as-proba**
-- *AssemblyScript transformers for unit testing and reading a file* with **as-transformer**
+- *Most essential Classes and helper functions* with **as-types**
```
Comment (Ben-Rey) [inline README.md:9]:
```suggestion
- *Essential Classes and helper functions* with **as-types**
```

**Review by Ben-Rey — CHANGES_REQUESTED** [review]

File: README.md (hunk 7 above)
Comment (Ben-Rey) [reply README.md]: I think now it is the constructor that is called automatically, so I am not sure if this statement is still true. I may be wrong

**Review by Ben-Rey — APPROVED** [review]

File: README.md (hunk 7 above)
Comment (Elli610) [reply README.md]: oh yes

**Pull request merged**